policyengine_uk/parameters/gov/dwp/housing_benefit/allowances/single/aged.yaml

```diff
@@ -0,0 +1,13 @@
+description: The following personal allowance amount is provided to single filers over the pension age under the Housing Benefit.
+values:
+  2015-06-01: 181.00
+  2021-04-01: 191.15
+  2022-04-06: 197.10
```

Comment:
[Could we add the latest values (up to April 2024) from this Commons Library report?](https://commonslibrary.parliament.uk/research-briefings/cbp-9872/#:~:text=Inflation%2Dlinked%20benefits%20and%20tax,single%20people%20aged%20under%2025)

Reply:
Added but I would want to consolidate with the legislative source down the road